### DISCUSSION TITLE
Issue with renewing github student pack

### DISCUSSION THREAD

**Discussion by xsafter** [community/community#38008] [GitHub Education]
Hello, I probably have a problem with the extension of the Github Student Pack. My current license ends on November 2nd. I would like to renew the license in advance, as it will take some time to review it, and I need products from the student pack now. If I want to apply for an extension, it says that `Hi, xsafter! You were last verified as a student on Nov 2, 2021. It is not necessary for you to re verify at this time.`
Is this ok?

**Comment by xsafter** [comment]
Update: my license expired yesterday, but I still can't activate it
![Screenshot 2022-11-03 at 11 27 07-p](https://user-images.githubusercontent.com/64601545/199676281-ab04ced3-24d3-4445-83c8-5d5f419e5b39.png)

**Comment by ImLilDev** [comment]
same issue ! you find a solution yet ?

**Reply by xsafter** [reply]
Oh,it's just that my license was activated for 2 years and I forgot about it :)

**Reply by samus-aran** [reply]
Hi @ImLilDev 👋 

As I mentioned a student renewal usually happens every 2 years depending on the studying. Does that information help you with your situation?

**Comment by samus-aran** [comment]
Hi @xsafter 👋 

Based on your screenshots it appears that you don't require to renew your student status for another year. Usually a student status is kept for 2 years. In your case your next renewal is 2023 🙂

**Reply by xsafter** [reply]
Thanks 🙃
